### PR TITLE
 Implementing Mish : A Self Regularized Non-Monotonic Neural Activation Function

### PR DESCRIPTION
**Pull request by MythicArrow**
This activation function was proposed in the BMVC2020 conference, and it contains several advantages over other activation functions, which are improved performance: Mish has been shown to outperform other activation functions such as ReLU, Leaky ReLU, and Swish on various benchmarks. Reduced vanishing gradients: Mish helps to reduce the vanishing gradient problem, which can occur when using ReLU or other activation functions. Self-gating: The Mish function has a self-gating property, which means that it can adapt to the input data and adjust its output accordingly. And its definition is f(x) = x* tanh(softplus(x)).
https://www.bmvc2020-conference.com/assets/papers/0928.pdf (This was the version that was proposed at the conference.)
https://arxiv.org/vc/arxiv/papers/1908/1908.08681v1.pdf (arXiv version)
https://github.com/digantamisra98/Mish (GitHub repo link)

### CODE REVIEW

**Comment by hawkinsp** [comment]
Already present in JAX: https://docs.jax.dev/en/latest/_autosummary/jax.nn.mish.html#jax.nn.mish
